backend/GoogleAPI.ts

```diff
@@ -7,33 +8,44 @@ const PATH_TO_KEY_FILE = process.env.GOOGLE_SERVICE_ACCOUNT_JSON_FILE;
 
 const SPREADSHEET_ID = process.env.SPREADSHEET_ID;
 
-const AUTHENTICATION = new google.auth.GoogleAuth({
-  keyFile: PATH_TO_KEY_FILE,
-  scopes: SCOPES,
-});
-
-const SHEETS = google.sheets({
-  version: 'v4',
-  auth: AUTHENTICATION,
-});
-
```

Comment:
Siirsin nämä kaksi variablea funktion sisään. Syy tähän on se, että laitoin varmuuden vuoksi `try/catch`-blokin sisään. Ennen `try/catch`in käyttöä heitti koodi melko mielenkiintoisen virhekoodin frontin toasteihin, jos oli virheellinen JSON-tyyli
![Screenshot from 2024-10-21 20-22-58](https://github.com/user-attachments/assets/4144e2de-6856-40f7-b5d6-e2e160fdee1b)

Reply:
Hyvä muutos 👍